config/prod.exs

```diff
@@ -12,9 +13,83 @@ use Mix.Config
 config :mindwendel, MindwendelWeb.Endpoint,
   cache_static_manifest: "priv/static/cache_manifest.json"
 
+secret_key_base =
+  System.get_env("SECRET_KEY_BASE") ||
+    raise """
+    Environment variable SECRET_KEY_BASE is missing.
+    You can generate one by calling: mix phx.gen.secret
+    """
+
+url_host =
+  System.get_env("URL_HOST") ||
+    raise """
+    Environment variable URL_HOST is missing.
+    The URL_HOST should be the domain name (wihtout protocol and port) for accessing your app.
+    """
+
+config :mindwendel, MindwendelWeb.Endpoint,
+  url: [host: url_host, port: 80],
+  http: [
+    port: String.to_integer(System.get_env("PORT") || "4000"),
+    transport_options: [socket_opts: [:inet6]]
+  ],
+  secret_key_base: secret_key_base
+
 # Do not print debug messages in production
 config :logger, level: :info
 
+#
+# Configuring the mindwendel repo
+#
+# If the env variable `DATABASE_URL` is set,
+# we will us this to configure the repo endpoint.
+#
+database_url = System.get_env("DATABASE_URL")
+
+if database_url do
+  Logger.info("Environment variable DATABASE_URL is defined and used for Mindwendel.Repo")
+  config :mindwendel, Mindwendel.Repo, url: database_url
```

Comment:
pool size is missing here

Reply:
`pool_size` defaults to 10, see https://hexdocs.pm/ecto/Ecto.Repo.html . Furthermore, the pool_size can be set in the url via query params. So this is not a problem.